### PR TITLE
Fix strawberry-graphql union error. 

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Strawberry-graphql made a recent change to the way it handles [strawberry.union](https://github.com/strawberry-graphql/strawberry/blob/main/CHANGELOG.md#01910---2023-06-28). This change is considered breaking because users who have the latest kedro-viz but haven't updated their strawberry-graphql package will encounter the following error:

```bash
TypeError: union() missing 1 required positional argument: 'types'
```

To fix this we have bumped strawberry-graphql dependency package to be at least version 0.192. 

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1441"></a>

### CODE REVIEW

**Comment by rashidakanchwala** [comment]
@tynandebold - yeah we upgraded mypy 1.0 and so unpinned the strawberry-version which was previously pinned to only 0.180 and lower. Also updated the code due to strawberry-graphql Deprecation warnings.

File: package/requirements.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ watchgod~=0.8.2
 plotly>=4.0
 pandas>=0.24
 sqlalchemy>=1.4, <3
-strawberry-graphql>=0.99.0, <1.0
+strawberry-graphql>=0.191.0, <1.0
```
Comment (astrojuanlu) [inline package/requirements.txt:12]: `0.191` doesn't exist right? https://pypi.org/project/strawberry-graphql/#history

Comment (rashidakanchwala) [reply package/requirements.txt]: updated to 0.192

**Review by astrojuanlu — APPROVED** [review]
LGTM!

**Review by merelcht — APPROVED** [review]
LGTM!

**Review by tynandebold — APPROVED** [review]

**Pull request merged**